### ISSUE TITLE
[Question]Harvester virtual machine failure

### ISSUE THREAD

**Issue by wangch1030** [harvester/harvester#5399]
## question
- A virtual machine in the Harvester 1.2.1 cluster cannot enter the operating system
![image](https://github.com/harvester/harvester/assets/144816721/7a1b1116-132e-4b8b-b93c-17a9acbd41f7)
## findings
This virtual machine uses the Kirin system, and data is persisted through longhorn
We found through the history command that applications such as ZenDAS, Gitlab, Java microservices, nginx, nacos, and mysql8 are deployed in the virtual machine. The fault occurred at 7:40 and was fixed by xfs-repair before restarting the system at 10:28.
![image](https://github.com/harvester/harvester/assets/144816721/cf5b7cb7-ea20-4343-806a-9f65b8d05e54)
## analysis
- From the perspective of K8S, analyze the custom resource crd: virtualmachineinstance and find out that the corresponding PVC is mounted through the volueSnapshot. This is persisted after storing the class through longhorn, so the files will not be lost and are still stored in the virtual machine.
- According to the error message, files or folders such as xfs, xfslog, dmsag, etc. cannot be found after entering the virtual machine, and relevant information cannot be found from the perspective of the virtual machine.
- Migrate the virtual machine to the host (from node4 to node11). Based on the analysis of the system logs of the two hosts, it was found that there are a large number of logs with Raw-Read-ErrorMate disk read and write error rates. Due to the damage to the xfs file system, the disk read and write error rate is high. The reason for the damage to the xfs file system has not yet been found.

We are unable to find out why the virtual machine malfunctioned.

**Comment by Yu-Jack** [comment]
Hi @wangch1030, could you provide the support bundle file by following this tutor https://docs.harvesterhci.io/v1.2/troubleshooting/harvester#generate-a-support-bundle? Another question is that are there same virtual machine acted like this issue? Thanks!

**Comment by wangch1030** [comment]
Hi,@Yu-Jack,I encountered a timeout issue while operating the support bundle
![e4a490e4bd1ea6100636344310bd4cdd_](https://github.com/harvester/harvester/assets/144816721/c87c2394-fd3b-4952-acd1-c17227b0b6bf)
![939f5b6b6627f50f4bec0a22183899c7_](https://github.com/harvester/harvester/assets/144816721/4cfd0645-9b5b-4db7-96b5-bd6009da4a55)
![20dbbda28f4b1201bdfcbbd002737b09_](https://github.com/harvester/harvester/assets/144816721/dd133f84-0d70-4409-8045-bed554452893)
- There are no other machines experiencing this situation. 
- After repairing or rebuilding the virtual machine, we redistributed the corresponding applications. Currently, all virtual machines are working properly. 
We look forward to understanding the causes of the problem and taking preventive measures to reduce the occurrence of similar situations.Thanks
